src/main/java/core/basesyntax/service/processor/ServiceProcessorOfShopImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.processor;
+
+import core.basesyntax.model.FruitRecord;
+import core.basesyntax.service.strategy.TypeStrategy;
+import java.util.List;
+
+public class ServiceProcessorOfShopImpl implements ServiceProcessorOfShop {
+    private TypeStrategy typeStrategy;
+
+    public ServiceProcessorOfShopImpl(TypeStrategy typeStrategy) {
+        this.typeStrategy = typeStrategy;
+    }
+
+    @Override
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the method definition.